infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -521,10 +521,8 @@ public static void restartServer(ServerContext serverContext, boolean resetData)
     /**
      * Attempt to cleanly shutdown all the servers.
      */
-    public static void cleanShutdown(@Nonnull NettyServerRouter router) {
+    public static void cleanShutdown(@Nonnull List<AbstractServer> servers) {
```

Comment:
It seems to me that use `NettyServerRouter` is better than a list of abstract servers. What was the reason, why you changed it?